components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -7774,24 +7773,43 @@ public String addComment(APIIdentifier identifier, Comment comment, String user)
      * Returns a specific comment of an API
      *
      * @param commentId  Comment ID
-     * @param identifier API identifier
+     * @param apiTypeWrapper API type wrapper
      * @return Comment Array
      * @throws APIManagementException
      */
-    public Comment getComment(APIIdentifier identifier, String commentId) throws APIManagementException {
+    public Comment getComment(ApiTypeWrapper apiTypeWrapper, String commentId) throws APIManagementException {
 
         Comment comment = new Comment();
         Connection connection = null;
         ResultSet resultSet = null;
         PreparedStatement prepStmt = null;
+        Boolean isProduct = apiTypeWrapper.isAPIProduct();
```

Comment:
Use primitive boolean

Reply:
Resolved in 529ba9909a41f6a98475171e424cb0a0782d19c5